src/coreclr/inc/strongnameinternal.h

```diff
@@ -15,16 +15,6 @@ typedef struct {
     BYTE         PublicKey[1];   // variable length byte array containing the key value in format output by CryptoAPI
 } PublicKeyBlob;
 
-// Determine the number of bytes in a public key
-DWORD StrongNameSizeOfPublicKey(const PublicKeyBlob &keyPublicKey);
-
-bool StrongNameIsValidPublicKey(_In_reads_(cbPublicKeyBlob) const BYTE *pbPublicKeyBlob, DWORD cbPublicKeyBlob);
-bool StrongNameIsValidPublicKey(const PublicKeyBlob &keyPublicKey);
-
-// Determine if a public key is the ECMA key
-bool StrongNameIsEcmaKey(_In_reads_(cbKey) const BYTE *pbKey, DWORD cbKey);
-bool StrongNameIsEcmaKey(const PublicKeyBlob &keyPublicKey);
-
 HRESULT StrongNameTokenFromPublicKey(BYTE* pbPublicKeyBlob,  // [in] public key blob
```

Comment:
StrongName token is always 8 bytes (SN_SIZEOF_TOKEN). We can delete StrongNameFreeBuffer and get callers to provide 8 byte buffer instead.